packages/desktop-client/src/components/Modals.tsx

```
@@ -589,6 +594,42 @@
           );
 
+        case 'edit-access':
+          return (
+            <EditUserAccess
+              key={name}
+              defaultUserAccess={options.access}
+              onSave={options.onSave}
+            />
+          );
+
+        case 'edit-access':
```

Biome

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)